internal/api/attestationconfig/azure.go

```diff
@@ -34,14 +34,14 @@ type AzureSEVSNPVersion struct {
 	Microcode uint8 `json:"microcode"`
 }
 
-// AzureSEVSNPVersionGet is the request to get the version information of the specific version in the config api.
-type AzureSEVSNPVersionGet struct {
+// AzureSEVSNPVersionAPI is the request to get the version information of the specific version in the config api.
+type AzureSEVSNPVersionAPI struct {
```

Comment:
Why did you rename this? And what is the rationale behind the name? 
The previous name seemed pretty intuitive to me and follows the pattern of other objects within the Resource API.

Reply:
The Resource can be used for upload and get. Do u have a better naming suggestion?

Reply:
Makes sense :+1: Maybe adapt the comment?
I guess it should just be named `AzureSEVSNPVersion`. But we already have that. I guess we can fix that when refactoring this to use `AttestationCfg`.